From qhfeng.kernel@gmail.com  Mon Nov 17 16:35:17 2008
From: Qinghuang Feng <qhfeng.kernel@gmail.com>
Date: Fri, 14 Nov 2008 20:24:05 +0800
Subject: USB: gadget: pxa27x_udc.c: cleanup kernel-doc
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: qhfeng.kernel@gmail.com
Message-ID: <491d6ded.0e0d6e0a.4656.7c38@mx.google.com>


no argument named @index in pio_irq_disable, and
no argument named @req in inc_ep_stats_bytes,
remove them.

Signed-off-by: Qinghuang Feng <qhfeng.kernel@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/pxa27x_udc.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/gadget/pxa27x_udc.c
+++ b/drivers/usb/gadget/pxa27x_udc.c
@@ -430,7 +430,6 @@ static void pio_irq_enable(struct pxa_ep
 /**
  * pio_irq_disable - Disables irq generation for one endpoint
  * @ep: udc endpoint
- * @index: endpoint number
  */
 static void pio_irq_disable(struct pxa_ep *ep)
 {
@@ -586,7 +585,6 @@ static void inc_ep_stats_reqs(struct pxa
  * inc_ep_stats_bytes - Update ep stats counts
  * @ep: physical endpoint
  * @count: bytes transfered on endpoint
- * @req: usb request
  * @is_in: ep direction (USB_DIR_IN or 0)
  */
 static void inc_ep_stats_bytes(struct pxa_ep *ep, int count, int is_in)
